wordpress-mobile/WordPress-iOS
# Feature/reader x posts

This PR adds new styling and behavior for cross-posts that we receive in certain reader feeds. 

To test:
- View a feed that contains cross posts. 
- Confirm that the cross post cell content matches the appearance of cross post cells on the web. (Note that the author name may appear slightly different and this is expected.)
- Confirm that tapping the cross post cell takes you to the source post's detail, not the detail of the cross post. 

Needs review: @kwonye
